### ISSUE TITLE
Create accessibility tooltips to Buttons on Home Screen

### ISSUE THREAD

**Issue by blueedgetechno** [blueedgetechno/quizmaster#2]
Missing accessibility tooltips on Home Screen Button

- Change Theme Button
- Create Quiz Button
- Profile Picture
